manifests/core/type/hub/storage/neo4j.yaml

```diff
@@ -32,7 +32,7 @@ spec:
           "acceptValue": {
             "$id": "#/properties/acceptValue",
             "type": "boolean",
-            "const": false
+            "const": true
```

Comment:
why do we need this change?

Reply:
That was an issue I spotted - According to the [proposal](https://github.com/capactio/capact/blob/main/docs/proposal/20211207-delegated-storage.md): "specifies if a given storage backend (app) accepts TypeInstance value while creating/updating TypeInstance, or just context". So in case of neo4j yes, we do accept value 🙂